assets/scss/buttons-links.scss

```diff
@@ -14,6 +14,14 @@
     background: $blue;
     box-shadow: 2px 2px 6px 0 rgba(0, 0, 0, 0.15);
   }
+
+  &.btn-cta {
+    margin: 2rem auto;
+    padding: 0.5rem 1rem;
+    border-radius: $border-radius-small;
+    box-shadow: $boxshadow-light;
+    text-decoration: none;
+  }
```

Comment:
Suggestion (in combination with the other suggestion below): 
```
&.btn-cta {
    margin-bottom: 2rem;
    margin-top: 2rem;
    padding: 0.5rem 1rem;
    text-decoration: none;
  }
```

Reply:
sure this makes sense